sys/net/sock/tls/Makefile.include

```diff
@@ -0,0 +1,5 @@
+INCLUDES += -I$(RIOTBASE)/sys/net/sock/tls/include
+INCLUDES += -I$(PKGDIRBASE)/wolfssl
```

Comment:
There seems to be a mismatch between the module dependencies and the include paths: wolfssl is not modelled as a dependency but is added to the include path. If WolfSSL is just one option to choose as a backend for the TLS implementation, it shouldn't be added here at all.